src/pkg/utils/io.go

```diff
@@ -171,6 +171,10 @@ func ReplaceTextTemplate(path string, mappings map[string]*TextTemplate, depreca
 // the skipPermission flag can be provided to ignore unauthorized files/dirs when true
 // the skipHidden flag can be provided to ignore dot prefixed files/dirs when true
 func RecursiveFileList(dir string, pattern *regexp.Regexp, skipPermission bool, skipHidden bool) (files []string, err error) {
+	if pattern == nil {
+		pattern = regexp.MustCompile(`(?mi).+$`)
```

Comment:
Most regex's don't apply outside of their functions. If there are any used across multiple functions then I would say yes, but since this is localized to this function I think it makes less sense.